src/dataflow-types/src/types.rs

```diff
@@ -417,11 +437,18 @@ pub struct ProtobufEncoding {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct SourceDesc {
     pub connector: SourceConnector,
-    /// Optionally, filtering and projection to optionally apply.
+    /// Optionally, filtering and projection to apply.
```

Comment:
This was correct as originally written, fwiw. It may contain a linear operator, and if it does there is no requirement to apply it (only, the option to apply it if it would help).

Reply:
Thanks, adjusted to a more verbose form of the original (which really read as a typo to my ear).